### DISCUSSION TITLE
ubuntu24.04 6.8.0-48-generic驱动编译后无法使用ffmpeg转码

### DISCUSSION THREAD

**Discussion by zz541843** [Icarusradio/intel-media-driver-dg1#32] [Q&A]
首先，肯定一下作者的教程文档，很清晰，一步步跟下来完成了驱动安装

![image](https://github.com/user-attachments/assets/a190206e-a868-4da6-beca-d1f450567470)
![image](https://github.com/user-attachments/assets/c82987d2-c43a-4da0-96e8-86121851a66e)

ffmpeg用hevc_qsv转码报错
![image](https://github.com/user-attachments/assets/d11107df-8398-4c82-a7a3-536bf86f8129)

水平有限，没试过docker里的jellfin，现在就只是想在ubuntu上ffmpeg调用dg1解码，我也试过从头编译DKMS，一样的报错

**Comment by zz541843** [comment]
还是不死心，安装了22.04，这次按照 [https://dgpu-docs.intel.com/driver/installation.html#ubuntu](url)的流程走了一遍，ffmpeg可以转码hevc了，但是av1不行，在24.04上试了也是一样
然后按照你的教程，从22.04执行apt install --install-recommends linux-generic-hwe-22.04，开启xe驱动，在docker里的jellyfin开启硬件转码播放hevc的视频无法转码，但是直接从容器里的/usr/lib/jellyfin-ffmpeg/ffmpeg进行hevc_qsv测试转码的话是可以转码的，容器里的ffmpeg用av1_qsv也会报错
![image](https://github.com/user-attachments/assets/8f29cd54-7d64-4bfb-af00-7fd985c227c0)


难受

**Comment by zz541843** [comment]
又尝试了几次，无法解决，很诡异，看了很多直通jellyfin教程都是一样的
pve新装ubuntu22.04
执行apt install --install-recommends linux-generic-hwe-22.04
修改grub参数开xe
docker安装 nyanmisaka/jellyfin 
开启硬件加速QSV
无论播放什么视频都是直接播放，而且更诡异的是我把dg1换成直通uhd770也是这样，这jellyfin就是无法硬件解码，测试命令都是过的

**Comment by Icarusradio** [comment]
如果是按照我写的教程开启内核xe模块驱动，然后测试结果也都对的话，建议去问问特供版作者 nyanmisaka 大佬
我写的教程我都是自己试验过能够运行的，可能问题不是出在内核驱动这边

**Comment by nyanmisaka** [comment]
xe kmd 和 ffmpeg 不是给小白用的。jellyfin-ffmpeg 也不是普通的 ffmpeg。

创建 MFX session 失败是缺少 VPL 运行库。

DG1、UHD7xx 硬件不支持 AV1 编码器，只能解码。AV1 编码需要 ARC 显卡起步。

Jellyfin 直接播放是大多数用户的期望，客户端能解码还非要服务器转码。这不是没事找事么？

![image](https://github.com/user-attachments/assets/6d77c055-c517-418f-a43e-a0aec672a753)
